index.js

```diff
@@ -399,10 +456,12 @@ ko.bindingHandlers.fastForEach = {
         $context: context
       });
     }
+    if (FastForEach.DEBUG)
```

Comment:
OK, I'll try to be a good boy :) Actually I like omitting the curly brackets, but I'll take care in the future.